docs/developer-docs/integrations/ledger/ledger-local-setup.md

```diff
@@ -19,9 +19,9 @@ Follow the steps below to deploy your copy of the ledger canister to a local rep
 
     :::
 
-2.  Make sure you use a recent version of DFX. If you don’t have DFX installed, follow instructions on <https://smartcontracts.org/> to install it.
+2.  Make sure you use a recent version of DFX. If you don’t have DFX installed, follow instructions on the [build](../../build/install-upgrade-remove) section to install it.
```

Comment:
Same as above: not sure the section name is correct.

Reply:
changed